### ISSUE TITLE
compute-val-loss argument is ignored

### ISSUE THREAD

**Issue by AndreaPi** [fizyr/keras-retinanet#1011]
I'm using `keras_retinanet` in a Docker container. The Dockerfile is

```
FROM nvcr.io/nvidia/tensorflow:19.03-py3

LABEL maintainer="..."

ADD requirements.txt /tmp/requirements.txt

ADD startup.sh /

WORKDIR /

RUN apt-get -y update -o Acquire::https::Verify-Peer=false && apt-get -y install \
    libglib2.0-0 \
    libsm6 \
    libxext6 \
    libxrender1 \
    git

RUN pip3 install -r tmp/requirements.txt

RUN pip install --user git+https://github.com/fizyr/keras-retinanet.git

RUN pip3 install --user git+https://github.com/cocodataset/cocoapi.git#subdirectory=PythonAPI

CMD /startup.sh

```
In other words, I'm installing the repository from GitHub, so it should be the latest master, right? I train the model this way:

```
python3 -m keras_retinanet.bin.train --compute-val-loss --snapshot-path /gdxray-snapshots csv /data/GDXray/keras_retinanet_annotations/train_annotations.csv /data/GDXray/keras_retinanet_annotations/classes.csv
```

Notwithstanding the `--compute-val-loss` argument, the validation loss is not computed:

` 2627/10000 [======>.......................] - ETA: 21:53 - loss: 0.0436 - regression_loss: 0.0435 - classification_loss: 1.0193e-04
`

1.  how can I get the model to compute the validation loss?
2. how can I visualize the training history? Should I add the argument `--tensorboard-dir` to the training command?

**Comment by AndreaPi** [comment]
Just for completeness, the validation loss is not computed at the end of an epoch either:

```
10000/10000 [==============================] - 1780s 178ms/step - loss: 0.0444 - regression_loss: 0.0443 - classification_loss: 1.2491e-04

Epoch 00036: saving model to /gdxray-snapshots/resnet50_csv_36.h5
Epoch 37/50
```

**Comment by AndreaPi** [comment]
I found the solution, the correct CLI call is:

```
python3 -m keras_retinanet.bin.train --weights /gdxray-snapshots/resnet50_coco_best_v2.1.0.h5 --compute-val-loss --tensorboard-dir /ogs --snapshot-path /gdxray-snapshots csv  /data/GDXray/keras_retinanet_annotations/train_annotations.csv /data/GDXray/keras_retinanet_annotations/classes.csv --val-annotations /data/GDXray/keras_retinanet_annotations/test_annotations.csv
```

**Comment by mariaculman18** [comment]
I am working with the last version (0.5.1) and `--compute-val-loss` is either ignored or not recognize. Any clue what happened?
